### DISCUSSION TITLE
403 on rest api fetch

### DISCUSSION THREAD

**Discussion by danielegustoids** [VirtusLab/strapi-plugin-navigation#271] [Q&A]
Hello, 
first of all nice work! I really like your plugin.
Currently, I'm not able to fetch the menù end point (/api/navigation/render/1?type=TREE) i set up public permissions but stills 403
Any idea?
Thanks

**Comment by danielegustoids** [comment]
It seems that the rests need a full access token. a simple "read-only" is not enough.

**Reply by cyp3rius** [reply]
Documentation was updated for the plugin. It's possible with `Custom` access token as well. You don't need to provide Full Access.

https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation#token-based
